docs/docs/extending-mdx.server.mdx

```diff
@@ -81,6 +81,8 @@ See also the [list of remark plugins][remark-plugins] and
     — change frontmatter (YAML) metadata to exports
 *   [`goodproblems/remark-mdx-math-enhanced`](https://github.com/goodproblems/remark-mdx-math-enhanced)
     — enhance math with JavaScript inside it
+*   [`pangelani/remark-mdx-chartjs`](https://github.com/pangelani/remark-mdx-chartjs)
+    — replace fenced code blocks with charts using `react-chartjs-2`.
```

Comment:
I think it would be nice to add a link to `react-chartjs-2`.

```suggestion
    — replace fenced code blocks with charts using [`react-chartjs-2`](https://react-chartjs-2.js.org/).
```